db.php

```diff
@@ -289,8 +289,7 @@ function chooseMentor() {
 
 
 	 function getMentor($mentor) {
-		global $_USER;
-		$user = $_USER['uid'];
+		$user = $mentor;
```

Comment:
Whoops I misread what this does. I thought it got the specific user's mentor. This is actually getting the info for any mentor. I think it's fine the way it is.